### PR TITLE
Add BZRX space

### PR DESCRIPTION
**Pull request by tomcbean**
**Add space for BZRX**  
  
Etherscan: https://etherscan.io/address/0x56d811088235F11C8920698a204A5010a788f4b3
CoinGecko: https://coingecko.com/en/coins/bzx-protocol
Website: https://bzx.network
Twitter: https://twitter.com/bZxHQ

### CODE REVIEW

File: spaces/bzx/skin.scss [hunk 1]
```diff
@@ -0,0 +1,10 @@
+.name {
```
Comment (bonustrack) [inline spaces/bzx/skin.scss:1]: The class name should be the same as the space key
```suggestion
.bzx {
```
And the file should be stored in the repo root "/skins/bzx.scss". 
Also you should add a field "skin": "bzx" in the index.json file.

Comment (jxdv) [inline spaces/bzx/skin.scss:1]: Skin class name should be the same as your space key `bzx.scss`
Also should be added into `/skins/bzx.scss` folder and not `/spaces/bzx/bzx.scss`

**Comment by tomcbean** [comment]
The suggested fixes are made.

File: skins/bzx.scss [hunk 2]
```diff
@@ -0,0 +1,10 @@
+.bzx {
+  --primary-color: #384aff;
+  --bg-color: white;
+  --text-color: #586069;
+  --link-color: #111111;
+  --heading-color: #111111;
+  --border-color: #d1d5da;
+  --header-bg: white;
+  --block-bg: transparent;
+}
```
Comment (e-asia) [inline skins/bzx.scss:10]: If you use the default theme you will have to delete the scss file.

File: spaces/bzx/index.json [hunk 3]
```diff
@@ -0,0 +1,16 @@
+{
+  "key": "bzx",
+  "name": "bZx",
+  "skin": "bzx",
```
Comment (e-asia) [inline spaces/bzx/index.json:4]: If you use the default theme you will have to remove the skin field.

**Review by e-asia — CHANGES_REQUESTED** [review]

File: spaces/bzx/index.json (hunk 3 above)
Comment (tomcbean) [reply spaces/bzx/index.json]: done

**Pull request merged**